### DISCUSSION TITLE
Support for scheduled jobs

### DISCUSSION THREAD

**Discussion by dvlsadvc8** [linkedpipes/etl#969] [Q&A]
Is there a method to implement automated pipeline executions? Experimenting with a local installation and the demo site, it appears that pipelines are executed manually. Is there support for scheduling recuring pipeline execution? For example, recuring data loads from other services to check for new data, or loading from filesystem if new files have been dropped.

**Comment by jakubklimek** [comment]
There is [an API](https://github.com/linkedpipes/etl/wiki/Pipeline-execution-with-curl) that we use with `cron` to execute scheduled pipelines.
